tools/assert_version.py

```diff
@@ -31,12 +31,6 @@
 assert config['metadata']['version'] == str(python_version), \
     "python/setup.cfg package version is incorrect"
 
-binder_requirements = open('.binder/requirements.txt').readlines()
-for line in binder_requirements:
-    if line.startswith("opendp=="):
-        assert line == f"opendp=={python_version}\n", \
-            ".binder/requirements.txt opendp dependency is incorrect"
```

Comment:
If this PR doesn't move forward, should file an issue to make a fix here: `opendp[polars]` is now specified, so the match wouldn't catch what we expect it to.